scripts/deploy/08_steth_call_theta_vault.ts

```diff
@@ -16,14 +11,17 @@ const main = async ({
   const chainId = network.config.chainId;
 
   if (chainId === CHAINID.AVAX_MAINNET || chainId === CHAINID.AVAX_FUJI) {
-    console.log(`08 - Skipping deployment stETH Call Theta Vault on ${network.name} because no stEth on Avax`);
+    console.log(
+      `08 - Skipping deployment stETH Call Theta Vault on ${network.name} because no stEth on Avax`
+    );
     return;
   }
 
```

Comment:
will these even deploy? we don't add AUCTION_DURATION, PREMIUM_DISCOUNT params in init when they are clearly there https://github.com/ribbon-finance/ribbon-v2/blob/600b44648a40aed5e3ca06c2700410e0dfba66d7/contracts/vaults/STETHVault/RibbonThetaSTETHVault.sol#L149

Reply:
I don't think so, the script is broken.  I didn't test this deploy script, only fixed the linting errors

Reply:
I added PREMIUM_DISCOUNT and AUCTION_DURATION back however I still think this script is failing and I already spent too much time trying to fix it on Kovan with no luck and don't really see this as a priority right now.